README.md

```diff
@@ -115,6 +115,17 @@ Browsershot::html('Foo')
   ->setChromePath("/path/to/my/chrome")
 ```
 
+### Pass custom arguments to Chromium
+
+If you need to pass custom arguments to Chromium, use the `addChromiumArg` method.
+
+This can be useful to fix font rendering issues on some Linux distributions (e.g. CentOS).
+
+```php
+Browsershot::html('Foo')
+  ->addChromiumArg('--font-render-hinting=none');
```

Comment:
Could you rename the function to `addChromiumArguments`. Let it accept an array with key values, instead of passing a string. Don't make the user type `--` before an argument, prefix that automatically.

Reply:
Thanks for the feedback! Will make the requested changes tomorrow, all are sensible and make sense to me. 👍

Reply:
@freekmurze Have made the requested changes;

- Renamed function to `addChromiumArguments`
- Accepts an array with key/values, or simply values, or both.
- Automatically prefixes the arguments with the double dash (`--`).

Any other issues, let me know.